packages/docs/stories/components/checkbox.stories.ts

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable */
+
+/* eslint-disable import/no-relative-packages */
+
+import '../../../components/src/components/checkbox/checkbox';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { userEvent } from '@storybook/testing-library';
+import { waitUntil } from '@open-wc/testing-helpers';
+import docsTokens from '../../../tokens/src/figma-tokens/_docs.json';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../src/helpers/component.js';
+const { args, argTypes } = storybookDefaults('syn-checkbox');
+const { overrideArgs } = storybookHelpers('syn-checkbox');
+const { generateTemplate } = storybookTemplate('syn-checkbox');
+
+const meta: Meta = {
+  component: 'checkbox',
+  args,
```

Comment:
```suggestion
    args: overrideArgs(
    [
      { name: 'default', type: 'slot', value: 'Default' }
    ],
    args
  ),
```

It would be good to show an example.